solr/solr-ref-guide/modules/deployment-guide/pages/taking-solr-to-production.adoc

```diff
@@ -455,7 +455,7 @@ Ideally this would mean more machines, but budget constraints might make that im
 There is another issue once the heap reaches 32GB.
 Below 32GB, Java is able to use compressed pointers, but above that point, larger pointers are required, which uses more memory and slows down the JVM.
 
-Because of the potential garbage collection issues and the particular issues that happen at 32GB, if a single instance would require a 64GB heap, performance is likely to improve greatly if the machine is set up with two nodes that each have a 31GB heap.
+Because of the potential garbage collection issues and the particular issues that happen at 32GB, if a single instance would require a 64GB heap, performance is likely to improve greatly if the machine is set up with two nodes that each have a 32GB heap.
```

Comment:
Remove this section altogether. Replace it (optionally) with something like:
> If you need to go over 31GB of heap, then you need to consider multiple nodes since they typically will perform better than one node with >32GB of heap.